### PR TITLE
add periodic snake

### PR DESCRIPTION
**Pull request by maflaven**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,7 @@ A selection of major game studios, publishers, etc. using GitHub:
 * [Octocat Jump](https://github.com/ogoshen/game-off-2012) - GitHub Game Off 2012 Entry. [Play it now!](http://ogoshen.github.io/game-off-2012/)
 * [Onslaught Arena](https://github.com/lostdecade/onslaught_arena) - Fight off hordes of classic medieval monsters in this fast paced arcade shooter! [Play it now!](http://arcade.lostdecadegames.com/onslaught_arena/)
 * [Pappu Pakia](https://github.com/mindd-it/pappu-pakia) - Addictive game where you guide little Pappu around obstacles while collecting coins, stars, and berries and avoiding enemies known as Pakias. [Play it now!](http://khele.in/pappu-pakia/)
+* [Periodic Snake](https://github.com/ranginuitrot/periodic-snake.js) - It's a wave! [Play it now!](http://periodic-snake.herokuapp.com/)
```
Comment (leereilly) [inline README.md]: It doesn't look like there's anything deployed to http://periodic-snake.herokuapp.com. Is there another URL? :thinking:

Comment (ranginuitrot) [inline README.md]: There is! I moved it since the last commit. Updated.

File: README.md [hunk 2]
```diff
@@ -132,7 +132,8 @@ A selection of major game studios, publishers, etc. using GitHub:
 * [Octocat Jump](https://github.com/ogoshen/game-off-2012) - GitHub Game Off 2012 Entry. [Play it now!](http://ogoshen.github.io/game-off-2012/)
 * [Onslaught Arena](https://github.com/lostdecade/onslaught_arena) - Fight off hordes of classic medieval monsters in this fast paced arcade shooter! [Play it now!](http://arcade.lostdecadegames.com/onslaught_arena/)
 * [Pappu Pakia](https://github.com/mindd-it/pappu-pakia) - Addictive game where you guide little Pappu around obstacles while collecting coins, stars, and berries and avoiding enemies known as Pakias. [Play it now!](http://khele.in/pappu-pakia/)
-* [Periodic Snake](https://github.com/ranginuitrot/periodic-snake.js) - It's a wave! [Play it now!](http://periodic-snake.herokuapp.com/)
+* [Periodic Snake](https://github.com/ranginuitrot/periodic-snake.js) - It's a wave! [Play it now!](https://ranginuitrot.github.io/periodic-snake.js/)
```
Comment (ranginuitrot) [inline README.md]: I moved it since the last commit. Updated!

File: README.md (hunk 1 above)
Comment (ranginuitrot) [inline README.md]: I updated the URL in the latest merge.

**Pull request closed without merging**